bitbots_vision/bitbots_vision/vision_modules/yoeo_vision_components.py

```diff
@@ -0,0 +1,890 @@
+#! /usr/bin/env python3
```

Comment:
Do you include this shebang intentionally? I believe, this is not necessary as we do not want to execute this file standalone.

Reply:
Nope, didn't add it (and haven't noticed it so far). Maybe PyCharm added it...